pkgs/servers/xandikos/default.nix

```diff
@@ -27,10 +27,6 @@ python3Packages.buildPythonApplication rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  postInstall = ''
-    installManPage xandikos.1
-  '';
```

Comment:
Why was this removed? If it is no longer requires also remove the nativeBuildInput.

Reply:
Removed due to https://github.com/jelmer/xandikos/commit/1fc7e78af7ccf17d3b542833ccfea8074b509355